app/design/frontend/vector/theme1/etc/view.xml

```diff
@@ -8,18 +8,6 @@
 <view xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Config/etc/view.xsd">
     <media>
         <images module="Magento_Catalog">
-            <image id="bundled_product_customization_page" type="thumbnail">
```

Comment:
Hello, 
I tried your /etc/view.xml on my theme and product images disappeared in the list view:
<img src="http://dl3.joxi.net/drive/2018/11/07/0021/1588/1439284/84/6d666fd65e.png">
This is not a layout, Magento doesn't merge these etc/view.xml through the themes fallback. So please copy-paste back this original code. And keep in mind, only layout xmls are merged, bit not /etc/view.xml